src/button/index.tsx

```diff
@@ -41,10 +41,7 @@ const StyledButton = styled(Button)`
   }
 
   &:hover,
-  &:focus,
   &:active {
-    text-decoration: none;
```

Comment:
I would keep the text-decoration since i'm not sure why it's here in the first place.

Reply:
Yes same remark. And it's probably to avoid undeline on <a> tag.